spec/latest/core/index.html

```diff
@@ -560,6 +560,17 @@ <h3>Documenting a Web API</h3>
       <i>ApiDocumentation</i>.</p>
 
     <p class="issue">Describe the various properties of an operation.</p>
+
+    <p>Hydra allows also to document both <i>ApiDocumentation</i> and
+      hypermedia controls with human-readable descriptions by applying
+      <i>hydra:title</i> and <i>hydra:description</i> (as shown in the
+      previous examples above). The former states a name of such a
+      decorated element that could be displayed to a human, the latter
+      provides it's description to be presented to that human i.e. as a hint.</p>
```

Comment:
"Displayed to a human", "presented to a human". 

It's a bit repetitive. Let's make this paragraph more concise